extra/man/alacritty.5.scd

```diff
@@ -616,6 +616,11 @@ _action_ or a _command_.
 
 		The _enabled_ field controls if the hint should be underlined when
 		hovering over the hint text with all _mods_ pressed.
+	
+	*persist* <boolean>
+
+		Set to _false_ by default. When this is _true_, hints remain persistent 
+		after selection.
```

Comment:
Can we move this up to the other boolean options? I think that structures a little nicer.

Same goes for the order in the type definition of `enabled`.